fabtests/benchmarks/benchmark_shared.c

```diff
@@ -72,7 +72,9 @@ void ft_benchmark_usage(void)
 
 int pingpong(void)
 {
-	int ret, i;
+	int ret, i, inject_size;
+
+	inject_size = MIN(hints->tx_attr->inject_size, fi->tx_attr->inject_size);
```

Comment:
If no option is specified, won't hints->inject_size = 0?  If so, this disables the use of inject unless the caller specifies the option.

Reply:
good point. I think this needs to be combined with a flag.

Reply:
Updated.